packages/block-library/src/embed/embed-placeholder.native.js

```diff
@@ -40,12 +44,58 @@ const EmbedPlaceholder = ( {
 	);
 	const embedIconErrorStyle = styles[ 'embed__icon--error' ];
 
+	const cannotEmbedMenuPickerRef = useRef();
+
+	const errorPickerOptions = {
+		retry: {
+			id: 'retryOption',
+			label: __( 'Retry' ),
+			value: 'retryOption',
+			onSelect: () => {
+				tryAgain();
+			},
+		},
+		convertToLink: {
+			id: 'convertToLinkOption',
+			label: __( 'Convert to link' ),
+			value: 'convertToLinkOption',
+			onSelect: () => {
+				fallback();
+			},
```

Comment:
We could simplify this line by doing:
```suggestion
			onSelect: fallback,
```

Reply:
Agreed! a2dc77b